lib/index.js

```diff
@@ -13,6 +13,9 @@ module.exports = {
   get getMiddleware() {
     return require('./middleware');
   },
+  get DummyWatcher() {
```

Comment:
Not sure we want a `DummyWatcher` exposed here, it seems like its only used/needed internally when `--no-watch` is specified. In that case, we _probably_ don't need to export it here...

Reply:
okay